features/goscripts/cilium.go

```diff
@@ -0,0 +1,153 @@
+package features
```

Comment:
`features/goscripts/cilium.go ` -> `testrunner/cillium.go` and use a common package name. Discussion was ongoing on rocket.chat: https://chat.suse.de/group/caasp-qa-internal?msg=EzJhWSCSQtFS3HDDP

Reply:
I would not use `testrunner` to avoid confusion with `testrunner`, so maybe just `testrun` ?

Also I think we should follow https://github.com/golang-standards/project-layout so I suggest we use `internal/testrun`

Having a look `features/goscripts/cilium.go` I don't see anything specifically related to `cilium` so I don't see a reason to create yet a `cilium.go` or a `cilium` package.

Reply:
Moving `./features/goscripts/*` to `./internal/` is fine for the moment. All `/features/goscripts/*.go` files are in the `main` package.